src/Microsoft.Identity.Client/Platforms/net45/WindowsFormsWebAuthenticationDialogBase.cs

```diff
@@ -292,45 +292,39 @@ private void InitializeComponent()
                 : Screen.PrimaryScreen;
 
             // Window height is set to 70% of the screen height.
-            int uiHeight = (int) (Math.Max(screen.WorkingArea.Height, 160)*70.0/DpiHelper.ZoomPercent);
-            webBrowserPanel = new Panel();
-            webBrowserPanel.SuspendLayout();
+            int uiHeight = (int)(Math.Max(screen.WorkingArea.Height, 160) * 70.0 / DpiHelper.ZoomPercent);
+            _webBrowserPanel = new Panel();
+            _webBrowserPanel.SuspendLayout();
             SuspendLayout();
 
-            // 
             // webBrowser
-            // 
-            webBrowser.Dock = DockStyle.Fill;
-            webBrowser.Location = new Point(0, 25);
-            webBrowser.MinimumSize = new Size(20, 20);
-            webBrowser.Name = "webBrowser";
-            webBrowser.Size = new Size(UIWidth, 565);
-            webBrowser.TabIndex = 1;
-            webBrowser.IsWebBrowserContextMenuEnabled = false;
-
-            // 
+            _webBrowser.Dock = DockStyle.Fill;
+            _webBrowser.Location = new Point(0, 25);
+            _webBrowser.MinimumSize = new Size(20, 20);
+            _webBrowser.Name = "_webBrowser";
+            _webBrowser.Size = new Size(UIWidth, 565);
+            _webBrowser.TabIndex = 1;
+            _webBrowser.IsWebBrowserContextMenuEnabled = false;
+
             // webBrowserPanel
-            // 
-            webBrowserPanel.Controls.Add(webBrowser);
-            webBrowserPanel.Dock = DockStyle.Fill;
-            webBrowserPanel.BorderStyle = BorderStyle.None;
-            webBrowserPanel.Location = new Point(0, 0);
-            webBrowserPanel.Name = "webBrowserPanel";
-            webBrowserPanel.Size = new Size(UIWidth, uiHeight);
-            webBrowserPanel.TabIndex = 2;
-
-            // 
+            _webBrowserPanel.Controls.Add(_webBrowser);
+            _webBrowserPanel.Dock = DockStyle.Fill;
+            _webBrowserPanel.BorderStyle = BorderStyle.None;
+            _webBrowserPanel.Location = new Point(0, 0);
+            _webBrowserPanel.Name = "_webBrowserPanel";
```

Comment:
See above [comment](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/265#discussion_r108065306).